File: java/string-1/makeTags.java
/* The web is built with HTML strings like "<i>Yay</i>" which draws Yay as 
 * italic text. In this example, the "i" tag makes <i> and </i> which surround 
 * the word "Yay". Given tag and word strings, create the HTML string with 
 * tags around the word, e.g. "<i>Yay</i>".
 */
public String makeTags(String tag, String word) {
    return "<" + tag + ">" + word + "</" + tag + ">";
}

File: python/string-1/make_tags.py
# The web is built with HTML strings like "<i>Yay</i>" which draws Yay as 
# italic text. In this example, the "i" tag makes <i> and </i> which surround 
# the word "Yay". Given tag and word strings, create the HTML string with tags 
# around the word, e.g. "<i>Yay</i>".
def make_tags(tag, word):
    return "<" + tag + ">" + word + "</" + tag + ">"
